docs/api/associations/belongs-to-many.md

```diff
@@ -10,8 +10,8 @@ When the join table has additional attributes, these can be passed in the option
 UserProject = sequelize.define('user_project', {
   role: Sequelize.STRING
 });
-User.belongsToMany(Project, { through: UserProject });
-Project.belongsToMany(User, { through: UserProject });
+User.belongsToMany(Project, { through: 'UserProject' });
+Project.belongsToMany(User, { through: 'UserProject' });
```

Comment:
The change you made were incorrect in the scenario since there has been a model defined which is being references.
Please keep issues seperate, open an issue if you've found a bug with `foreignKey` in belongsToMany